meson.build

```diff
@@ -2,6 +2,7 @@ project('io.elementary.settings-daemon',
     'c', 'vala',
     version: '1.3.0',
     license: 'GPL3',
+    meson_version: '>=0.61.0'
```

Comment:
No longer needs 0.61?

Reply:
I'm not too sure. There is https://github.com/elementary/triage/issues/175 which recomends to set minimum meson version to 0.61.2. I think we should follow it.

Reply:
Hmmm, I think at least 0.58 features are used here so the distro affected in my mind are CentOS 8 Stream, openSUSE Leap 15.4, Slackware 15.0, I think I can give ACK if no Pantheon users are there... :woman_shrugging:

Reply:
Thanks, I lowered required version to 0.58 just in case.